vizro-core/docs/pages/user_guides/navigation.md

```diff
@@ -228,6 +228,8 @@ The same configuration for [grouping pages](#grouping-your-pages) applies inside
 
 You can alter the icons used by specifying the name of the icon in the [Google Material icons library](https://fonts.google.com/icons):
 
+*Note: Spaces in the icon's name are replaced with underscores*
```

Comment:
Even better, why don't we just do the conversion for them? We can easily add something to `NavLink` that does `str.lower().replace(" ", "_")` and then it will work straight away.

Reply:
The more I think about this, the more I think that the preferred way for a user to enter `icon` should actually be the human-readable icon name like `Double Arrow` rather than the computer-readable snakecase version `double_arrow`.

Please could you change our docs and tests to reflect this? I think just a find and replace of `icon=` should do it. e.g. in docs instead of saying `icon="bar_chart"` we should do `icon="Bar Chart"`.

If a user enters `bar_chart` or `bar chart` etc. it will still work fine, but `Bar Chart` should be the first thing we suggest they try rather than `bar_chart`.

Reply:
Ah, yes, I like that! Will implement

Reply:
Implemented in this commit: https://github.com/mckinsey/vizro/pull/190/commits/afe2968ec316b25a64d91126f6340d7a410e8086

Reply:
I just left a couple of comments on that commit (not sure if you can see the conversation anywhere else 🤔) but looks good!

Reply:
I've incorporated them in :)